### PR TITLE
Optimization CRD wait on manifest appyl

### PR DESCRIPTION
**Pull request by howardjohn**
Right now we spend ~5s waiting for CRDs to be applied. Typically these
aren't changing. Instead, we can skip the wait if we know no changes
were made.

### CODE REVIEW

File: operator/pkg/manifest/installer.go [hunk 1]
```diff
@@ -544,12 +544,33 @@ func objectsNotInLists(objects object.K8sObjects, lists ...object.K8sObjects) ob
...
+}
+
+func waitForCRDs(objects object.K8sObjects, stdout string, dryRun bool) error {
 	if dryRun {
 		scope.Info("Not waiting for CRDs in dry run mode.")
 		return nil
 	}
 
+	if canSkipCrdWait(stdout) {
+		scope.Info("Skipping CRD wait")
```
Comment (ayj) [inline operator/pkg/manifest/installer.go]: The reason why the check was skipped would be helpful.

**Review by ayj — APPROVED** [review]
Approved. I added a /hold to give an opportunity to address (or ignore) the comment on logging.

**Comment by howardjohn** [comment]
I agree, I improved the log and removed the hold

**Pull request merged**